modules/phase_field/tests/phase_field_kernels/tests

```diff
@@ -60,4 +60,11 @@
     input = 'ConstructionOrder.i'
     exodiff = 'ConstructionOrder_out.e'
   [../]
+
+  [./SimpleSplitCHWRes]
+    type = 'Exodiff'
+    input = 'SimpleSplitCHWRes.i'
+    exodiff = 'SimpleSplitCHWRes_out.e'
+    max_parallel = 1                
```

Comment:
I removed the max_parrallel and petsc options. I use -p 2 for the test, it seems fine. In this case, should I create a new pull request in next step? Thanks.